supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -402,7 +402,10 @@ async def update_access_token_payload(
         new_access_token_payload: Dict[str, Any],
         user_context: Dict[str, Any],
     ) -> bool:
-        """DEPRECATED: Use merge_into_access_token_payload instead"""
+        deprecated_warn(
+            "update_access_token_payload is deprecated. Use merge_into_access_token_payload instead"
+        )
```

Comment:
this should not come in here cause other functions that are not deprecated might be calling this? We should instead log this in the user facing function for this in asyncio/__init__.py file?

Reply:
done.